src/instagram.js

```diff
@@ -101,8 +101,7 @@ export async function apiInstagramPosts({
       const results = []
       results.push(...response.data.data)
       while (
-        response.data.paging.next ||
-        (maxPosts && results.length <= maxPosts)
+        maxPosts ? (response.data.paging.next && results.length <= maxPosts) : response.data.paging.next
```

Comment:
can you also please add a comment about this logic so its easier to get back to it? 
otherwise looking great and thanks!

Reply:
Ok, I will.

Reply:
Hi, @oorestisime ,

I have added some comments at there. Could you please review it? :)